docs/guides/distribution/updater.md

```diff
@@ -2,220 +2,128 @@
+- **macOS**: On macOS, Tauri will create a `.tar.gz` archive from the application bundle inside the `target/release/bundle/macos/` folder:
 
-ERROR is emitted when there is an error with the updater. We suggest listening to this event even if the dialog is enabled.
+  - `myapp.app` - the standard app bundle.
+  - `myapp.app.tar.gz` - the updater bundle.
+  - `myapp.app.tar.gz.sig` - the signature of the update bundle.
 
-#### Rust
+- **Linux**: On Linux, Tauri will create a `.tar.gz` archive from the AppImage inside the `target/release/bundle/appimage/` folder:
 
-```rust
-window.listen("tauri://update-status".to_string(), move |msg| {
-  println!("New status: {:?}", msg);
-})
-```
+  - `myapp.AppImage` - the standard app bundle.
+  - `myapp.AppImage.tar.gz` - the updater bundle.
+  - `myapp.AppImage.tar.gz.sig` - the signature of the update bundle.
 
-#### Javascript
+- **Window**: On Windows, Tauri will create a `.zip` archive from the MSI installer inside the `target/release/bundle/msi/` folder:
+  - `myapp.msi` - the standard app bundle.
+  - `myapp.msi.zip` - the updater bundle.
+  - `myapp.msi.zip.sig` - the signature of the update bundle.
 
-```js
-import { listen } from '@tauri-apps/api/event'
-listen('tauri://update-status', function (res) {
-  console.log('New status: ', res)
-})
-```
+The signature can be uploaded and shared safely as long as your private key is secure.
 
 ## Server Support
 
-Your server should determine whether an update is required based on the [Update Request](#update-requests) your client issues.
+Tauri's updater supports 2 kinds of servers. You can host a static JSON file on services like S3 or GitHub gists, or host a dynamic update server. The static JSON file is easier to use while an update server will give you finer control over the update mechanism.
+
+### Dynamic Update Server
 
-If an update is required, your server should respond with a status code of [200 OK] and include the [update JSON](#update-server-json-format) in the body.
+With this approach, Tauri will follow the update server's instructions. To disable the internal version check you can [overwrite Tauri's version comparison] to always install the version sent by the server. This could be useful if you need to roll back your app version quickly.
 
-If no update is required your server must respond with a status code of [204 No Content].
+Your server can use variables defined in the `endpoint` url above to determine if an update is required. If you need more data, you can include additional [request headers in Rust] to your liking.
 
-### Update Server JSON Format
+Your server should respond with a status code of [`204 No Content`] if there is no update available.
 
-When an update is available, Tauri expects the following schema in response to the update request provided:
+If an update is required, your server should respond with a status code of [`200 OK`] and a JSON response in this format:
```

Comment:
Is this expected version documented anywhere else? Or is this the only place?

Reply:
Only here i think. The updater guide is somewhat of the source of truth for the server side stuff.